docs/TRACEBACKS.md

```diff
@@ -0,0 +1,200 @@
+# Reading and Troubleshooting Stack Traces in Python
+
+## Frame Object
+
+A frame object holds the local variables and arguments passed to a function.
+A frame object is created when a function is called and is destroyed when that function returns.
```

Comment:
This starts with the new concept (frame) vs with the background/context (when a function is called).

```suggestion
When a function is called, a frame object is created to hold the local variables and arguments passed to a function.
When the function is returns, the frame object is destroyed.
```